engineeringstores/react-stores-2022
# 18 implement shopify buy button backend

Closes #18 

## **Requirements:**

- [x] Create Shopify buy button component
- [x] Implement Shopify services within the component

### To Test:

- Run frontend and backend by running `yarn start:dev`
- Navigate to `Product/6716470722769`
- Product buy button should be present
- Cart should function when add to cart is clicked

### Resources Used:

- [Shopify BuyButton.js Docs](https://shopify.github.io/buy-button-js/customization/) 
- [Shopify BuyButton.js Github](https://github.com/Shopify/buy-button-js) 
- [Shopify StoreFront API Examples (React Buy)](https://github.com/Shopify/storefront-api-examples/blob/master/react-js-buy/src/App.js) 
- [Shopify JavaScript Buy SDK Github](https://github.com/Shopify/js-buy-sdk)
